documentation/asciidoc/computers/camera/libcamera_apps_packages.adoc

```diff
@@ -30,4 +30,4 @@ These will be installed automatically when needed.
 
 * `libepoxy-dev` contains the necessary `libepoxy` header files and resources. You will only need this if you want support for the X11/GLES preview window.
 
-Subsequently `libcamera-apps` can be xref:camera.adoc#building-libcamera-apps-without-rebuilding-libcamera[checked out from Github and rebuilt].
+Subsequently `libcamera-apps` can be xref:camera_software.adoc#building-libcamera-apps-without-rebuilding-libcamera[checked out from Github and rebuilt].
```

Comment:
Minor quibble, but I think the correct capitalisation is 'GitHub' ?

Reply:
Will fix!